tests/restapi/test_statistics.py

```diff
@@ -33,7 +33,11 @@ def linearize_namespace(tree_namespace, linear_namespace=None):
     return linear_namespace
 
 
+# TODO: Not sure why this fails?
```

Comment:
Same reason as the `tests/orm.test_querybuilder.py::test_statistics*` tests failing. This test calls the `statistics` endpoint which calls that function and that uses `date_trunc` which is not implemented by sqlite